src/Tests/VideoBundleTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\media\Tests;
+
+use Drupal\simpletest\WebTestBase;
+
+/**
+ * Ensures that media bundle for videos can be created.
+ *
+ * @group media
+ */
+class VideoBundleTest extends WebTestBase {
```

Comment:
PHPCodeSniffer finds few things in this file: 

`
➜  media git:(1f42b36) phpcs --standard=Drupal --extensions=php,module,inc,install,test,profile,theme,js,css,info,txt,md src/Tests/VideoBundleTest.php    

FILE: ...http/drupal8/docroot/modules/media/src/Tests/VideoBundleTest.php
----------------------------------------------------------------------
FOUND 12 ERRORS AND 2 WARNINGS AFFECTING 13 LINES
----------------------------------------------------------------------
 14 | ERROR   | [x] Line indented incorrectly; expected 2 spaces,
    |         |     found 4
 19 | ERROR   | [x] Line indented incorrectly; expected 2 spaces,
    |         |     found 4
 19 | ERROR   | [ ] If the line declaring an array spans longer than
    |         |     80 characters, each element should be broken into
    |         |     its own line
 21 | ERROR   | [x] Line indented incorrectly; expected 2 spaces,
    |         |     found 4
 26 | ERROR   | [x] Line indented incorrectly; expected 2 spaces,
    |         |     found 4
 28 | ERROR   | [x] Line indented incorrectly; expected 2 spaces,
    |         |     found 4
 31 | ERROR   | [x] Line indented incorrectly; expected 2 spaces,
    |         |     found 4
 32 | ERROR   | [x] Line indented incorrectly; expected 4 spaces,
    |         |     found 8
 33 | ERROR   | [x] Line indented incorrectly; expected 4 spaces,
    |         |     found 8
 40 | ERROR   | [x] Line indented incorrectly; expected 6 spaces,
    |         |     found 8
 41 | WARNING | [x] A comma should follow the last multiline array
    |         |     item. Found: 'field_video'
 46 | WARNING | [x] A comma should follow the last multiline array
    |         |     item. Found: 'field_source'
 55 | ERROR   | [x] Expected 1 blank line after function; 0 found
 56 | ERROR   | [x] The closing brace for the class must have an
    |         |     empty line before it
----------------------------------------------------------------------
PHPCBF CAN FIX THE 13 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 53ms; Memory: 6Mb
`

We should probably check existing files too and fix notices in a separate pull request. CodeSniffer docs: https://www.drupal.org/node/1419988